tests/PHPUnit/System/expected/test_TwoVisitors_twoWebsites_differentDays_VisitsSummary.get_firstSite_lastN__API.getProcessedReport_day.xml

```diff
@@ -44,10 +44,10 @@
 	</columns>
 	<reportData>
 		<result prettyDate="Sunday 3 January 2010">
-			<nb_uniq_visitors>1</nb_uniq_visitors>
+			<nb_uniq_visitors>2</nb_uniq_visitors>
 			<nb_users>0</nb_users>
-			<nb_visits>1</nb_visits>
-			<nb_actions>1</nb_actions>
+			<nb_visits>2</nb_visits>
+			<nb_actions>2</nb_actions>
 			<max_actions>1</max_actions>
 			<nb_actions_per_visit>1</nb_actions_per_visit>
 			<avg_time_on_site>00:00:00</avg_time_on_site>
```

Comment:
It's all good. There is one visit at '2010-01-03 12:22:33', one visit at '2010-01-03 23:55:00' and another one for the next day at '2010-01-04 00:01:00'.

With the old behaviour it was one visit for '2010-01-03 12:22:33', and one visit together for '2010-01-03 23:55:00' and '2010-01-04 00:01:00'. The visit counted for Monday before as the last_action_time was on Monday.